library/db/src/main/java/org/keynote/godtools/android/db/GodToolsDatabase.java

```diff
@@ -98,6 +100,8 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     case 40:
                         db.execSQL(ToolTable.SQL_V40_ALTER_OVERVIEW_VIDEO);
                         break;
+                    case 41:
+                        db.execSQL(GlobalActivityAnalyticsTable.SQL_V41_CREATE_GLOBAL_ANALYTICS);
```

Comment:
this will cause a crash when a user upgrades from v40 to v41, why?

Reply:
this is separate from my other comment about long term issues with the `SQL_V41_CREATE_GLOBAL_ANALYTICS` constant

Reply:
I forgot the break right?

Reply:
yeah